python/README.md

```diff
@@ -0,0 +1,25 @@
+# Python Arrow Flight Client Exmaple
+This lightweight Python client connects to the Dremio Arrow Flight service endpoint. This client requires the username and password for authentication. The hostname and port used is default to `localhost` and `32010`, these settings can be changed by providing the hostname and port as arguments when running the client. Moreover, the tls option can be provided to establish an encrypted connection. Please note that trusted certificates must be provided when the tls option is enabled. This client can query any datasets in Dremio that are accessible by the Dremio user provided.
```

Comment:
Capitalize 'Service' since it part of a product/feature name, similar to how 'Service is capitalized in 'Amazon Elasticsearch Service'.